web/src/js/ducks/flows.js

```diff
@@ -123,6 +123,7 @@ export function update_content(flow, file) {
         file = new Blob([file], {type: 'plain/text'})
     body.append('file', file)
     fetchApi(`/flows/${flow.id}/response/content`, {method: 'post',  body} )
+    update(flow, {response: {headers: [['Content-Encoding', '']]} } )
```

Comment:
thats the problem with gzip, i have to disable it, otherwise i got an error. should i gzip on my own? any hints ?